addon/-private/system/record-array-manager.js

```diff
@@ -76,6 +76,13 @@ export default class RecordArrayManager {
     this.internalModelDidChange(internalModel);
   }
 
+  recordWasUnloaded(internalModel) {
+    let array = this._liveRecordArrays[internalModel.modelName];
```

Comment:
@dwickern Since `array` isn't changing, maybe it's best to use `const` here?

Reply:
It was to match the existing code style

Reply:
Ah, I see. :+1: